lib/drizzlepac/createMedian.py

```diff
@@ -168,16 +168,22 @@ def _median(imageObjectList, paramDict):
         native_units = image.native_units
         if lthresh is not None:
             if proc_units.lower() == 'native':
-                if native_units.lower() == "counts":
+                if native_units.lower() == 'counts':
```

Comment:
I would suggest changing this to

    if native_units.lower().startswith('counts'):

because regardless of whether the units are 'counts/s' or 'counts', you should multiply by the gain.  The same comment applies to line 180.

Reply:
Yes, we have to deal with "counts" separately (i.e., multiply by gain) but I think I still want to have one long if..elif...else instead of splitting it into two blocks because I want to get to the last `else` and raise an exception when `native_units` are not recognized as it happened when `WFC3` image class was created with `native_units` set to `'ELECTRONS/S'` but this code was not updated with these new units (because it never crashed).